server/controllers/bookings/getAllBookings.js

```diff
@@ -6,18 +6,18 @@ const book = new Book();
 const getAllBookings = async (req, res) => {
   try {
     const getUser = jwt.decode(req.headers.authorization);
-    if (getUser.isAdmin === true) {
-      const allBookings = await book.getAtllbookings;
+    if (getUser.isadmin === true) {
```

Comment:
Why did you change this (`isAdmin`) from being a camelCase?
You can also better write the above with just `if (getUser.isadmin)`